README.md

```diff
@@ -19,3 +19,50 @@ When a request is forwarded from Kong it adds a few headers when the app-key has
 2. X-Consumer-Custom-ID: The custom_id of the Consumer (if set) --> This is only set on system-consumers (e.g. the learningpath-frontend consumer)
 3. X-Consumer-Username: The username of the Consumer (if set) --> This is set on all personal users, but not on system-consumers.
 
+## Getting Access Tokens
+
+Tokens are obtained from the ```/auth/tokens``` endpoint.  Before you can get access tokens, you first need to obtain 
+client credentials (a client id and a client secrect) that are specific to the API and operations that you want access to.
+To get your client credentials contact us. 
```

Comment:
Kanskje vi skulle hatt noe mer enn `us`? Det kommer kanskje en e-postadresse på et senere tidspunkt for slikt?